### PR TITLE
Feature/stackn 174

### PR DESCRIPTION
**Pull request by stefanhellander**
## Status

- [X] Ready
- [ ] Draft
- [ ] Hold

## Description

Create jobs from command line.
stackn create job -c job.json
stackn get jobs

Example of job definition:
{
  "command": "ls /home/app/",
  "environment": "default-gpu",
  "schedule": "None",
  "resources.limits.cpu": "1000m",
  "resources.limits.memory": "2Gi",
  "resources.requests.cpu": "1000m",
  "resources.requests.memory": "2Gi",
  "resources.gpu.enabled": "true",
  "resources.requests.gpu": "1"
}

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [X] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [ ] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [ ] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

File: components/chart-controller/controller/controller.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ def deploy(self, options, action='install'):
             return json.dumps({'status':'failed', 'reason':'Option release not set.'})
 
         args = ['helm', action, '--kubeconfig', kubeconfig, options['release'], chart]
-
+        # args.append('--dry-run')
```
Comment (dstoyanova) [inline components/chart-controller/controller/controller.py:50]: Could be removed perhaps?

File: components/studio/experiments/models.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from django.db import models
 from django.db.models.signals import pre_delete, pre_save
 from django.conf import settings
-from api.serializers import ProjectSerializer
+# from api.serializers import ProjectSerializer
```
Comment (dstoyanova) [inline components/studio/experiments/models.py:4]: I think we can remove the import if it's not used anymore.

File: components/studio/experiments/models.py [hunk 3]
```diff
@@ -42,7 +43,25 @@ def pre_save_experiments(sender, instance, using, **kwargs):
 
     user_config_file = create_user_settings(instance.username)
     user_config_file = yaml.dump(json.loads(user_config_file))
-
+    # print(settings.NAMESPACE)
```
Comment (dstoyanova) [inline components/studio/experiments/models.py:46]: This could be removed as well?

**Review by dstoyanova — APPROVED** [review]

File: components/chart-controller/controller/controller.py (hunk 1 above)
Comment (stefanhellander) [reply components/chart-controller/controller/controller.py]: Done

File: components/studio/experiments/models.py (hunk 3 above)
Comment (stefanhellander) [reply components/studio/experiments/models.py]: Done

File: components/studio/experiments/models.py (hunk 2 above)
Comment (stefanhellander) [reply components/studio/experiments/models.py]: Done

**Pull request merged**